projects/igniteui-angular/src/lib/splitter/splitter-pane/splitter-pane.component.ts

```diff
@@ -102,7 +103,13 @@ export class IgxSplitterPaneComponent {
 
     /** @hidden @internal */
     @HostBinding('style.order')
-    public order!: number;
+    public get order() {
+        return this._order;
+    }
+    public set order(val) {
+       this._order = val;
+       this.cdr.markForCheck();
```

Comment:
perhaps markForCheck only if the value does change?

Reply:
BTW, if you use a `signal()` for `_order` you should get automatic mark for check.